main.cpp

```diff
@@ -352,6 +379,17 @@ void FishTank::feedFish(int aquariumSize)
         std::cout << "feed fish at least three times a week" << std::endl;
     }        
 }
+
+struct FishTankWrapper
+{
+    FishTankWrapper(FishTank* FishTankPointer) : fishTankptr(FishTankPointer){}
+    ~FishTankWrapper()
+    {
+        delete fishTankptr;
+    }
+    
+    FishTank* fishTankptr = nullptr;
```

Comment:
`camelCase` your member variable name.  
Type names begin with capital letters.  variables do not.
- [ ] revise